data/strings/strings.txt

```diff
@@ -16326,6 +16326,234 @@
+    sw = 4. Washa Huduma za Mahali
+    th = 4. เปิดบริการระบุตำแหน่ง
+    tr = 4. Konum Servislerini Açın
+    uk = 4. Увімкніть Служби визначення місцезнаходження
+    vi = 4. Bật Dịch vụ định vị
+    zh-Hans = 4.打开定位服务
+    zh-Hant = 4.開啟定位服務
+
+  [location_services_disabled_on_device_additional_message]
+    tags = ios
+    en = Or continue using Organic Maps without Location
+    af = Of gaan voort om Organic Maps sonder ligging te gebruik
+    ar = أو استمر في استخدام الخرائط العضوية بدون الموقع
+    az = Və ya Məkan olmadan Üzvi Xəritələrdən istifadə etməyə davam edin
+    be = Або працягвайце выкарыстоўваць Organic Maps без месцазнаходжання
+    bg = Или продължете да използвате Organic Maps без местоположение
+    ca = O continuar utilitzant Organic Maps sense ubicació
+    cs = Nebo pokračujte v používání aplikace Organic Maps bez polohy
+    da = Eller fortsæt med at bruge Organic Maps uden placering
+    de = Oder verwende Organic Maps weiterhin ohne Standort
+    el = Ή συνεχίστε να χρησιμοποιείτε τους Organic Maps χωρίς Τοποθεσία
+    es = O seguir utilizando Organic Maps sin localización
+    es-MX = O continuar usando Organic Maps sin Ubicación
+    et = Või jätkake Organic Maps kasutamist ilma asukohata
+    eu = Edo jarraitu kokapenik gabeko Organic Maps erabiltzen
+    fa = یا به استفاده از نقشه های ارگانیک بدون موقعیت مکانی ادامه دهید
+    fi = Tai jatka Organic Maps käyttöä ilman sijaintia
+    fr = Ou continuer à utiliser Organic Maps sans Location
+    he = או המשך להשתמש במפות אורגניות ללא מיקום
+    hi = या स्थान के बिना ऑर्गेनिक मानचित्र का उपयोग जारी रखें
+    hu = Vagy folytassa az Organic Maps használatát helymeghatározás nélkül
+    id = Atau lanjutkan menggunakan Organic Maps tanpa Lokasi
+    it = Oppure continua a utilizzare le Organic Maps senza la localizzazione
+    ja = または、位置情報なしでオーガニック・マップを使い続ける
+    ko = 또는 위치 없이 오가닉 맵을 계속 사용하세요.
+    lt = Arba toliau naudokite "Organic Maps" be vietos
+    mr = किंवा स्थानाशिवाय ऑरगॅनिक नकाशे वापरणे सुरू ठेवा
+    nb = Eller fortsett å bruke Organic Maps uten plassering
+    nl = Of blijf Organic Maps gebruiken zonder locatie
+    pl = Lub kontynuuj korzystanie z Organic Maps bez lokalizacji
+    pt = Ou continua a utilizar o Organic Maps sem Localização
+    pt-BR = Ou continue usando o Organic Maps sem o Location
+    ro = Sau continuați să utilizați Organic Maps fără Locație
+    ru = Или продолжите использование Organic Maps без определения местоположения
+    sk = Alebo pokračujte v používaní Organic Maps bez polohy
+    sv = Eller fortsätt använda Organic Maps utan Plats
+    sw = Au endelea kutumia Organic Maps bila Mahali
+    th = หรือใช้แผนที่ทั่วไปโดยไม่มีตำแหน่งต่อไป
+    tr = Veya Konum olmadan Organic Maps kullanmaya devam edin
+    uk = Або продовжуйте використовувати Organic Maps без визначення місцезнаходження
+    vi = Hoặc tiếp tục sử dụng Bản đồ không phải trả tiền mà không có Vị trí
+    zh-Hans = 或继续使用 "有机地图 "而不定位
+    zh-Hant = 或繼續使用沒有位置的有機地圖
```

Comment:
Is it possible to extract these translations from GMaps iOS app? :)

Reply:
@biodranik in GMaps there are no such label :) 
I've added it to not confuse the user that without location the OM is useless (as gmaps - their offline mode is really awkward for me).